web/app/js/components/Navigation.jsx

```diff
@@ -4,16 +4,11 @@ import ArrowDropDownIcon from '@material-ui/icons/ArrowDropDown';
 import Badge from '@material-ui/core/Badge';
 import BreadcrumbHeader from './BreadcrumbHeader.jsx';
 import Button from '@material-ui/core/Button';
-import ChevronLeftIcon from '@material-ui/icons/ChevronLeft';
```

Comment:
Note: I pulled out the sidebar code into a `const` called `drawer` which is why so many lines are changed. Most of the diff lines are because of slightly different tabbing due to restructuring of file.